content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Please remove the blank line at the start of file.

Reply:
Done